raiden_contracts/data/source/services/OneToN.sol

```diff
@@ -34,7 +34,7 @@ contract OneToN is Utils {
     event Claimed(
         address sender,
         address indexed receiver,
-        uint256 expiration_block,
+        uint256 expiration_timestamp,
```

Comment:
Reminder for the naming scheme application (new file). Sorry to be very verbose with these comments. 🙈

Reply:
To be fixed in https://github.com/raiden-network/light-client/issues/3015